README.md

```diff
@@ -37,7 +43,24 @@ The easiest way to try the web app is to run it through docker:
 - npm
 - HDF5 and NetCDF4 headers, if available on your platform
 
-On Debian, e.g., install these packages:
+
+### MacOS
+You can install these dependencies using brew. Don’t know what [brew](http://brew.sh/) is?
```

Comment:
OK, so perhaps just say `You can install these dependencies using [brew](http://brew.sh).`